Client/fetch.js

```diff
@@ -0,0 +1,17 @@
+//Fetch data from API and display onto HTML
```

Comment:
Could this have just been in the script.js file,
can reuse variables and trim down the lines a bit that way